core/java/iesi-core/src/main/java/io/metadew/iesi/metadata/configuration/script/ScriptLabelConfiguration.java

```diff
@@ -39,7 +41,8 @@ private ScriptLabelConfiguration() {
     public Optional<ScriptLabel> get(ScriptLabelKey scriptLabelKey) {
         try {
             String queryScriptLabel = "select ID, SCRIPT_ID, SCRIPT_VRS_NB, NAME, VALUE from " + getMetadataRepository().getTableNameByLabel("ScriptLabels")
-                    + " where ID = " + SQLTools.getStringForSQL(scriptLabelKey.getId()) + ";";
+                    + " where ID = " + SQLTools.getStringForSQL(scriptLabelKey.getId()) +
+                    " and DELETED_AT = 'NA' ;";
```

Comment:
Setting DELETED_AT to NA should not have an effect as ScriptLabelKey contains only a UUID